CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing to forte-music/web
+
+Thanks for taking the time to contribute to forte. Here are some guidelines to follow when contributing.
+
+Some of the code in this project hasn't been refactored to follow these guidelines yet.
+
+## CSS
+
+For the most part, CSS is written using [styled-components]. Currently, they are scattered near the presentational components they are used in, in `src/components/styled` and in `src/components`. Going forward, mixins should be placed in `src/styled-mixins`, shared styled components in `src/components/styled` and unshared styled components, near where they are used.
+
+Avoid using theming unless needed. They add un-needed complexity for most use cases. If you are using the same component with slightly different styles in two different places, then use themeing.
+
+## Redux
+The [render-props] pattern is used to decouple data providing components with components consuming data. We use a wrapper around `connect` (a Higher Order Component from `react-redux`) to use the render props pattern with redux. This wrapper is called `createReduxComponent` and can be found in `src/redux/render.tsx`.
+
+## Component Folder Structure
+There are a few different types of components.
+
+* Presentational Components
+  Stateless components which are primarily concerned with how things look. State and actions are passed as props and prop callbacks respectively. These are usually functional components. Usually placed in `src/components/`.
+
+* Enhancer Components
+  Often handle data fetching and state storage concerns. Usually stored next to the container component they are used in (for example `src/components/AlbumsContainer/enhancers/redux.ts`).
+
+* Container Components
+  Made up by composing many enhancers and on or more presentational components. Found in folders ending with `Container` (for example `src/components/AlbumsContainer/index.tsx`).
+
+[Here is some more information about the presentational and container component pattern.][container-component]
+
+## Code Style
+
+[Prettier] is used to keep our code formatted consistently. Run `yarn fix-style` to reformat code to follow adhere to Prettier's style. Run `yarn check-style` to see which files don't adhere to Prettier's style.
+
+## Storybook
+
+[Storybook] is used for testing react components outside of the complete application. Run `yarn storybook` to start it with mock data. Files ending with `.stories.tsx` are picked up storybook. A test ensures that all stories render without crashing.
+
+## Tests
```

Comment:
You have a "Linting" header, so this should be called "Testing", unless you want to change "Linting" to "Lints"